package/gluon-mesh-babel/luasrc/lib/gluon/upgrade/300-gluon-mesh-babel-mkconfig

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/lua
+-- TODO: Migrate this into uci
```

Comment:
do we really want to merge code that contains TODO statements?

Reply:
my two review-comments about the TODOs aren't saying what you have to do, only that you have to decide one way or the other and if you want to solve it later, remove the TODO and add a new issue to gluon about this rather than abusing comments for it